src/uu/ls/src/ls.rs

```diff
@@ -1825,7 +1825,7 @@ fn display_additional_leading_info(
             } else {
                 "?".to_owned()
             };
-            result.push_str(&format!("{} ", pad_left(&i, padding.inode)));
+            write!(result, "{} ", pad_left(&i, padding.inode)).unwrap();
```

Comment:
The `write!(...).unwrap()` calls seem okay, because `format!()` [seems to be infallible](https://doc.rust-lang.org/std/macro.format.html#panics), and `push_str()` does not return a `Result` anyway.